### DISCUSSION TITLE
Is a Mapbox API key required for the pydeck examples?

### DISCUSSION THREAD

**Discussion by keith555** [visgl/deck.gl#5401] [Q&A] [answered]
The pydeck [installation guide](https://pydeck.gl/installation.html) seems to say that a Mapbox API token is mandatory. I see that [#5079](https://github.com/visgl/deck.gl/pull/5079) replaced Mapbox with Carto in the deck.gl examples. When I run the python examples, I get a warning that no Mapbox API was supplied. Is it necessary for me to manually make the same change that was applied in #5079, and if so, how do I do that in pydeck?

**Comment by kylebarron** [comment]
What version of Pydeck are you using? I believe the most recent Pydeck release included the Carto basemaps support without an API key

**Comment by ajduberstein** [comment, marked as answer]
The Carto base-maps will appear in pydeck 0.6, which goes out this week. That should fix the issue with the warning.

**Reply by keith555** [reply]
Great. Thanks a lot.
